Workbooks/SymantecEndpointProtection.json

```diff
@@ -0,0 +1,809 @@
+        "size": 0,
+        "showAnalytics": true,
+        "title": "Total Events by User over Time",
+        "timeContext": {
+          "durationMs": 43200000
+        },
+        "timeContextFromParameter": "TimeRange",
+        "timeBrushParameterName": "SelectedTimeRange",
+        "showExportToExcel": true,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "categoricalbar"
+      },
+      "conditionalVisibility": {
+        "parameterName": "TabName",
+        "comparison": "isEqualTo",
+        "value": "RiskEvents"
+      },
+      "customWidth": "60",
+      "name": "Total Events by User over Time"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "SymantecEndpointProtection\n| where LogType == \"Agent Risk Logs\"\n| where CategorySet in ({RiskCategory}) or '*' in ({RiskCategory})\n| where RiskName in ({RiskNames}) or '*' in ({RiskNames})\n| where UserName in ({UserNames}) or '*' in ({UserNames})\n| where SrcIpAddr in ({IPAddress}) or '*' in ({IPAddress})\n| where SrcHostName in ({Hostnames}) or '*' in ({Hostnames})\n| summarize Total = count() by UserName\n| top 10 by Total",
+        "size": 0,
+        "showAnalytics": true,
+        "title": "Top 10 User Names",
+        "timeContext": {
+          "durationMs": 43200000
+        },
+        "timeContextFromParameter": "TimeRange",
+        "showExportToExcel": true,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "table",
+        "gridSettings": {
+          "formatters": [
+            {
+              "columnMatch": "Total",
+              "formatter": 8,
+              "formatOptions": {
+                "palette": "blue"
+              },
+              "numberFormat": {
+                "unit": 17,
+                "options": {
+                  "style": "decimal",
+                  "useGrouping": false
+                }
+              },
+              "tooltipFormat": {}
+            }
+          ]
+        }
+      },
+      "conditionalVisibility": {
+        "parameterName": "TabName",
+        "comparison": "isEqualTo",
+        "value": "RiskEvents"
+      },
+      "customWidth": "40",
+      "name": "Top 10 User Names"
+    }
+  ],
+  "fallbackResourceIds": [
+    "/subscriptions/7ac4b7c7-65f2-46c7-87bb-ca6ac8e5dbef/resourcegroups/socaas-rg/providers/microsoft.operationalinsights/workspaces/toncoso"
+  ],
+  "fromTemplateId": "sentinel-SymantecEndpointProtection",
```

Comment:
changes made

Reply:
@v-jayakal are these the correct changes?

Reply:
Yes.

Reply:
> 
> 
> @chicduong - There are conflicts in WorkbookMetadata.json, Can you please resolve them for approval and merging? Thanks.

@v-jayakal forgive me, I'm not seeing the conflicts in the WorkbookMetatdata.json. They seem to match the require content in the example